src/BumpInto.php

```diff
@@ -144,6 +151,19 @@ private static function updateLockContentHash(string $composerLockFile, string $
         $lockFile->write($lockData);
     }
 
+    private static function extractVersionRangeSign(string $version) : string
+    {
+        if (strpos($version, '~') === 0) {
+            return '~';
+        }
+
+        if (strpos($version, '^') === 0) {
+            return '^';
+        }
+
+        return '^';
```

Comment:
```suggestion
        return '';
```

Fixed constraints (like `1.2.3`) should not be changed.

Reply:
Good point! We do have tests for fixed constraints, and it is not reaching this piece of code ☺️